stdlib/csv.pyi

```diff
@@ -17,7 +17,7 @@ from _csv import (
     unregister_dialect as unregister_dialect,
     writer as writer,
 )
-from typing import Any, Generic, Iterable, Iterator, Mapping, Sequence, Type, TypeVar, overload
+from typing import Any, Collection, Generic, Iterable, Iterator, Mapping, Sequence, Type, TypeVar, overload
```

Comment:
One minor thing: We should include `Collection`, `Iterable`, `Iterator`, `Mapping`, and `Sequence` from `collections.abc` instead of `typing`.

Reply:
Whatever works best for you.

Reply:
f8af7bcf hopefully fixes this.

Just a thought -- would it be useful to have a flake8 plugin (custom to this repo) which checked for this? I suspect it would be fairly easy to create one and might make this preference more visible. If I recall correctly no-one would need to install anything extra, it would just be a file somewhere in the repo and some configuration in the `pyproject.toml` and flake8 would do the rest.

Reply:
@PeterJCLaw ambv/flake8-pyi#46 :)